src/hackerspace_online/adapter.py

```diff
@@ -16,6 +16,18 @@ def clean_username(self, username, shallow=False):
 class CustomSocialAccountAdapter(DefaultSocialAccountAdapter):
 
     def pre_social_login(self, request, sociallogin):
+        """
+        Invoked just after a user successfully authenticates via a
+        social provider, but before the login is actually processed
+        (and before the pre_social_login signal is emitted).
+
+        ---
+
+        Currently using this to automatically connect a google account
+        to an existing user provided that they used the same email address
+        for logging in
```

Comment:
The email address on the `User.email` is still important and what we will still be using.

I added new code so that every time User.email is changed, a confirmation email will be sent. Of course, they will be marked as unverified if they do so